package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mozilla/raptor",
-  "version": "3.3.1",
+  "version": "3.4.0",
```

Comment:
Let's remove the version bump. In thinking about it, changes to `devDependencies` that will only affect local development won't be impacted by versions, only the GH repo.